Sources/Hedera/QueryCost.swift

```diff
@@ -49,6 +49,8 @@ extension QueryCost: Execute {
 
     internal var explicitTransactionId: TransactionId? { nil }
 
+    internal var firstTransactionId: TransactionId? { nil }
+
```

Comment:
Does this do anything?

Reply:
@izik1 Now, this field serves to hold `initialTransactionId` in updated changes.